### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by grasshopper47**
Cleaned many, many methods

### CODE REVIEW

File: src/pages/_document.tsx [hunk 1]
```diff
@@ -1,23 +0,0 @@
-import NextDocument, {Head, Html, Main, NextScript} from "next/document";
-
-export default class Document extends NextDocument {
-    render() {
-        return (
-            <Html>
-                <Head>
-                    <title>NFTopt</title>
-                    <meta name="description" content="DappCamp Warriors is a tutorial project from DappCamp"></meta>
```
Comment (Stefaanna) [inline src/pages/_document.tsx:9]: ups :))

File: src/components/OptionDetailsPreview.tsx [hunk 2]
```diff
@@ -95,23 +95,38 @@ function OptionDetailsPreview(props: OptionDetailsPreviewProps) {
...
-        // Can exercise any time before & including the end day
-        if (option.flavor === OptionFlavor.AMERICAN) {
-            return isSameDay(today, endDate) || isBefore(today, endDate);
-        }
+        if (option.flavor === OptionFlavor.AMERICAN) { return isBefore(today, end_day); }
 
         return false;
     };
 
+    // const canExerciseOption = () => {
```
Comment (Stefaanna) [inline src/components/OptionDetailsPreview.tsx:108]: why

Comment (grasshopper47) [reply src/components/OptionDetailsPreview.tsx:108]: forgot, will remove

File: src/pages/_document.tsx [hunk 3]
```diff
@@ -1,23 +0,0 @@
-import NextDocument, {Head, Html, Main, NextScript} from "next/document";
```
Comment (Stefaanna) [inline src/pages/_document.tsx:1]: why delete this?

**Review by Stefaanna — APPROVED** [review]

**Pull request merged**